ste/sender-blockBlobFromURL.go

```diff
@@ -189,11 +189,12 @@ func (c *urlToBlockBlobCopier) generateStartCopyBlobFromURL(id common.ChunkID, b
 			c.jptm.FailActiveUpload("Pacing block", err)
 		}
 
-		_, err := c.destBlockBlobURL.CopyFromURL(ctxWithLatestServiceVersion, c.srcURL, c.metadataToApply,
-			azblob.ModifiedAccessConditions{}, azblob.BlobAccessConditions{}, nil, azblob.DefaultAccessTier, nil)
+		_, err := c.destBlockBlobURL.PutBlobFromURL(ctxWithLatestServiceVersion, azblob.BlobHTTPHeaders{}, c.srcURL, c.metadataToApply,
+			azblob.ModifiedAccessConditions{}, azblob.BlobAccessConditions{}, nil, nil, azblob.DefaultAccessTier, nil,
+			azblob.ClientProvidedKeyOptions{})
 
 		if err != nil {
-			c.jptm.FailActiveSend("Copy Blob from URL", err)
+			c.jptm.FailActiveSend("Put Blob from URL", err)
```

Comment:
Couple of things :-
1. There is a syntax error in testsuite folder (smoke tests). Please run `go build ./testSuite` to detect and fix that.
2. I see the default service version of AzCopy is `2019-12-12`.  `PutBlobFromUrl()` works with `2020-04-08`. Let's review and merge this [PR](https://github.com/Azure/azure-storage-file-go/pull/53) to update the service of file SDK to `2020-04-08`. After this PR is merged, we can raise default service version of AzCopy from `2019-12-12` to `2020-04-08`